custom.html.md.erb

```diff
@@ -143,6 +143,60 @@ $ cf create-buildpack BUILDPACK PATH POSITION [--enable|--disable]
 </pre>
 
 <%=vars.link_adminguid_buildpack%>
+### <a id='specifying-default-versions'></a> Specifying Default Versions ###
+As of `buildpack-packager` [version 2.3.0](https://github.com/cloudfoundry/buildpack-packager/releases/tag/v2.3.0) and later, it is possible to specify the default version for a given dependency. This is done by adding a `default_versions` object to the `manifest.yml` file.
```

Comment:
Speaking only for myself, "as of 2.3.0" is noise. It's why we have release notes.